vowpalwabbit/core/src/distributionally_robust.cc

```diff
@@ -108,11 +108,11 @@ ChiSquared& ChiSquared::update(double w, double r)
   return *this;
 }
 
-double ChiSquared::qlb(double w, double r)
+double ChiSquared::qlb(double w, double r, double sign)
```

Comment:
add comment here about the sign convention, just to be extra clear.

Reply:
updated